webapp/channels/src/packages/mattermost-redux/src/reducers/entities/threads/threadsInTeam.ts

```diff
@@ -283,6 +283,14 @@ export const unreadThreadsInTeamReducer = (state: ThreadsState['unreadThreadsInT
             return handleReceivedThread(state, action, extra);
         }
         return state;
+    case ThreadTypes.RECEIVED_THREADS:
```

Comment:
Looking at the code and at https://github.com/mattermost/mattermost-webapp/pull/8793 where `RECEIVED_UNREAD_THREADS` was added, I'm trying to think of why we didn't do this in the first case. Are there perhaps any cases where this might cause problems with saving or paving over something in the unreads tab that shouldn't be changed? I'm trying to think of situations like  loading the first page of of threads with an unread thread on the second page which might cause issues with it

Reply:
@devinbinnie can you add more context for @yasser on the QA Steps on what to check for possible regressions ?